File: templates/clojure.is-test/%kata_file%.clj
; Adapt the code to your code kata %kata_file%.
; Important: Test and production code has to be
;            completely in this file.

(use 'clojure.test)

(defn %kata_file% []
  "fixme")

(deftest %kata_file%-test []
	(is (= "foo" (%kata_file%))))

(run-tests)

File: templates/php.phpunit/%kata_file%.php
<?php
// Adapt the code to your code kata %kata_file%.
// Important: Test and production code has to be
//            completely in this file.

require_once 'PHPUnit/Autoload.php';

// Please do not rename the test class. If you change the name, you have to adapt it in run-once.sh.

class %Kata_file%Test extends PHPUnit_Framework_TestCase
{
    public function testFoo() {
        $objectUnderTest = new %Kata_file%();
        $this->assertEquals('foo', $objectUnderTest->foo());
    }
}

class %Kata_file%
{
    public function foo()
    {
        return 'fixme';
    }
}

